didx-xyz/yoma-mobile
# YOMA-471 - Fixing parsing error on some phones

![](https://media.giphy.com/media/jSAfAbR2bk0aPThxNJ/giphy.gif)
## Scope [YOMA-471](https://yomaworld.atlassian.net/browse/YOMA-471)
- fix install failing on some phones (parsing error)

## Work Done
- updated google-services.json
- went through unimodules install and redid all android project integrations
- renamed `beta` deploy in Fastfile to `alpha` deploy to keep consistent with alpha naming throughout
- removed all temp branches from circle builds.

### Additional:
- sorted package json scripts alphabetically and added a reset script

## Steps to Test
download version 5 from Android App Distribution and try install it (it should install)
